### DISCUSSION TITLE
Videoclip dump editing

### DISCUSSION THREAD

**Discussion by Blokador** [nesrak1/UABEA#279] [General]
Hi, I have a problem. Today I tried to replace original videoclip inside the game to my edited one by replacing some data in its dump file. After clicking ok button, the error appears when I try to edit file again before saving.
![image](https://github.com/nesrak1/UABEA/assets/136165417/bfd0f75a-477b-445d-b96b-5c4e82c6e9d7)
Some time later I noticed even after saving original dump of videoclip without any changes, it still modifies some data (to be precise, "FileSize m_Offset" and "UInt64 m_Size" in StreamedResource tab, which often take the values far of their boundaries). Also the variable "bool m_sRGB" is always set to false after saving, even if it's been true before.
![image](https://github.com/nesrak1/UABEA/assets/136165417/b991f6fc-120e-45c8-94dc-6d3cd5823334)
The game can't even play the original videoclip because after saving something modifies the data inexplicitly.

**Comment by nesrak1** [comment]
This was fixed a while ago but might have been after release 6. Try nightly.
